### PR TITLE
 Adds support for time_precision

### PR DESCRIPTION
**Pull request by SebastianThorn**

### CODE REVIEW

File: lib/influxdb/rails/configuration.rb [hunk 1]
```diff
@@ -93,6 +95,7 @@ def initialize
         @async              = DEFAULTS[:async]
         @use_ssl            = DEFAULTS[:use_ssl]
         @retry              = DEFAULTS[:retry]
+        @time_precision     = DEDAULTS[:time_precision]
```
Comment (dmke) [inline lib/influxdb/rails/configuration.rb:98]: There's a typo, it should be `DEFAULTS`.

**Review by dmke — CHANGES_REQUESTED** [review]

**Comment by dmke** [comment]
Looks good. Please correct the typo an squash the commits into one.

**Comment by SebastianThorn** [comment]
ye, never really got the hang of how these requests work.
:)

**Comment by dmke** [comment]
Which request in particular?

**Pull request closed without merging**